lsp/Main.java

```diff
@@ -0,0 +1,28 @@
+package lsp;
+
+public class Main {
+
+  /**
+   * Run a demo.
+   * @param args The ususal.
+   */
+  public static void main(String[] args) {
+    Rectangle r = getRectangle();
```

Comment:
Could you please rename the "r" variable so that the name is meaningful? We want the code to be readable.

Reply:
Good suggestion. Done!